### PR TITLE
Replace with libraries containing json1 support

### PR DESCRIPTION
**Pull request by Serdans**
Built from https://github.com/brodybits/Android-sqlite-ext-native-driver and https://github.com/liteglue/Android-sqlite-connector.

#69

### CODE REVIEW

File: react-native.config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
 				project: './platforms/ios/SQLite.xcodeproj'
 			},
 			android: {
-				sourceDir: './platforms/android'
+				sourceDir: './platforms/android-native'
```
Comment (andpor) [inline react-native.config.js]: so why are you changing this to android-native? ISn't this going to break for people wanting to use regular Android ?

Comment (Serdans) [reply react-native.config.js]: We were using android-native in our project. I'll change it back to android.

**Comment by andpor** [comment]
@Serdans  - i have a basic SQL test javascript in the module. Could you create a new one in the same directory for testing JSON1 functionality with the native modules ?

**Comment by Serdans** [comment]
@andpor added the JSON1 tests.

**Comment by andpor** [comment]
Published as part of 5.0.0
Thank you!

**Comment by punksta** [comment]
Nice work!
I am wondering, why building from sources, not using 
https://github.com/requery/sqlite-android ?

**Comment by andpor** [comment]
@punksta what do you mean exactly ?

**Comment by punksta** [comment]
@andpor I mean there is a popular and supported library which provides sqlite binaries for android. 

why you guys are using
https://github.com/brodybits/Android-sqlite-ext-native-driver and
https://github.com/liteglue/Android-sqlite-connector
instead?

**Comment by andpor** [comment]
@andpor because this is how this library is designed.

**Comment by FatemeMirzaeii** [comment]
> Published as part of 5.0.0
> Thank you!

can we use json1 extension now in version 5 of react-native-sqlite-storage?
if yes,how?

**Comment by andpor** [comment]
@FatemeMirzaeii - try running the JSON demo file.

**Comment by FatemeMirzaeii** [comment]
@andpor could you plz address the demo file?

**Comment by andpor** [comment]
@FatemeMirzaeii who would like me to address it ? It is there in the repo.

**Comment by naushad-madakiya** [comment]
Here is the list of JSON queries which are supported from 5.0.0, this is not available as default in Android but available with this library.

https://github.com/andpor/react-native-sqlite-storage/pull/392/files#diff-cfd439444497b2507bfb70ecc8ff7a78R161

**Comment by FatemeMirzaeii** [comment]
@andpor  @naushad-madakiya thanks.

**Comment by panda0603** [comment]
I am trying to use json_extract function in android, but I get error "message": "no such function: JSON_EXTRACT (code 1 SQLITE_ERROR):
Same code seems to work fine in ios.
What may be the problem?

**Pull request merged**